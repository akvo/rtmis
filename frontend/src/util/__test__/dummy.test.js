import dummy from "../dummy";

const chartData = [
  {
    name: "Var A",
    stack: "Stack 1",
    value: 10,
  },
  {
    name: "Var B",
    stack: "Stack 2",
    value: 13,
  },
  {
    name: "Var C",
    stack: "Stack 2",
    value: 20,
  },
  {
    name: "Var D",
    stack: "Stack 1",
    value: 24,
  },
  {
    name: "Var E",
    stack: "Stack 2",
    value: 14,
  },
];

describe("util/dummy", () => {
  test("test if dummy has a chartData as an array of objects", () => {
    expect(dummy.chartData).toBeDefined();
    expect(dummy.chartData).toEqual(expect.arrayContaining(chartData));
    expect(dummy).toMatchSnapshot({
      chartData: expect.any(Array),
    });
  });
});
