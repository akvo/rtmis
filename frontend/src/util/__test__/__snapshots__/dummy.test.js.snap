// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`util/dummy test if dummy has a chartData as an array of objects 1`] = `
Object {
  "chartData": Any<Array>,
}
`;
